JS/script.js

```diff
@@ -0,0 +1,304 @@
+  </ul>
+  <div class="see-project">
+    <a id="id4" class="see-project-link" href="#">See Project</a>
+  </div>
+</div>
+</section>`;
+
+portfolio.innerHTML = projects;
+
+const proObjects = [
+  {
+    name: 'Tonic',
+    id: 'id1',
+    featuredImage: './images/project-img/project1.svg',
+    description:
+    `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, 
+    but also the leap into electronic typesetting, remaining essent`,
+    technologies: ['html', 'css', 'javaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    name: 'Tonic',
+    id: 'id2',
+    featuredImage: './images/project-img/project2.svg',
+    description:
+    `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, 
+    but also the leap into electronic typesetting, remaining essent`,
+    technologies: ['html', 'css', 'javaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    name: 'Tonic',
+    id: 'id3',
+    featuredImage: './images/project-img/project3.svg',
+    description:
+    `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, 
+    but also the leap into electronic typesetting, remaining essent`,
+    technologies: ['html', 'css', 'javaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    name: 'Tonic',
+    id: 'id3',
+    featuredImage: './images/project-img/project4.svg',
+    description:
+    `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, 
+    but also the leap into electronic typesetting, remaining essent`,
+    technologies: ['html', 'css', 'javaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+];
+
+const popupWindow = document.createElement('div');
+popupWindow.classList.add('popup-wind');
```

Comment:
- [ ] When the popup is clicked, only one data is popping up throughout. I would advise you to make it pop the very clicked project.
- [ ] Upon clicking it, it pops a blank space until I scroll. I would advise you to make it appear where it is clicked.

Reply:
Thank you.
Fixed.